Doc/library/string.rst

```diff
@@ -657,9 +657,15 @@ Nesting arguments and more complex examples::
 Template strings
 ----------------
 
-Templates provide simpler string substitutions as described in :pep:`292`.
-Instead of the normal ``%``\ -based substitutions, Templates support ``$``\
--based substitutions, using the following rules:
+Template strings provide simpler string substitutions as described in
+:pep:`292`.  The primary use case for template strings is for
+internationalization (i18n) since in those contexts, the more featured yet
+complex formatting languages for f-strings and ``str.format()`` can make
+translations more difficult to implement.  As an example of a library for i18n
```

Comment:
f-strings were originally touted for translations, but I did point out the inappropriateness of them for that purpose.   Maybe we don't mention any of them, and just say "other built-in string formatting features".